storage/quota/quota.go

```diff
@@ -149,9 +149,13 @@ func GetDefaultQuota(quotas map[string]string) string {
 // SetRootfsDiskQuota is to set container rootfs dir disk quota.
 func SetRootfsDiskQuota(basefs, size string, quotaID uint32) error {
 	overlayfs, err := getOverlay(basefs)
-	if err != nil || overlayfs == nil {
+	if err != nil {
 		return fmt.Errorf("failed to get lowerdir: %v", err)
```

Comment:
I do not think this line of code is good.

If err == nil and overlay == nil, the condition is true, but the error message is so bad : `failed to get lowerdir:`.

Reply:
Just give a error you want, what I fix is overlayfs will never be non-nil. I am not care about the origin error message.